docker/java_dropin_env_base/README.md

```diff
@@ -1,12 +1,12 @@
 # Java drop-in environment base image
-Repository name: **datarobot/java_dropin_env_base**  
-Latest date: 08-29-2020  
```

Comment:
please don't remove double whitespaces - it is Markdown newline

Reply:
Mm, I forget about that. Can I use a trailing backslash instead? It's more visible and doesn't conflict w. my IDE automatically trimming trailing whitespaces. https://spec.commonmark.org/0.29/#hard-line-breaks

```
Repository name: **datarobot/java-dropin-env-base\**
```

Reply:
yes!
looks fine

Reply:
Actually figured out how to stop my IDE from trimming whitespace in Markdown. So I've taken your original suggestion.